app/_locales/en/messages.json

```diff
@@ -3675,9 +3675,6 @@
     "message": "$1 is hosted on npm and $2 is this Snap’s unique identifier.",
     "description": "$1 is the snap name and $2 is the snap NPM id."
   },
-  "metamaskExtension": {
```

Comment:
we need to move this pairing in each language file

Reply:
@DDDDDanica it only exists in 2 language files (both in this PR) since this was recently added